nixos/tests/certmgr.nix

```diff
@@ -145,7 +162,12 @@ let
 
             systemd.services.nginx.wantedBy = lib.mkForce [ ];
 
-            systemd.services.certmgr.after = [ "cfssl.service" ];
+            systemd.services.certmgr.after = [
+              "cfssl.service"
+              # make sure nginx starts up first so we can check that certmgr
+              # restarts it
+              "nginx.service"
```

Comment:
Part of the test checks that nginx starts twice but in practice, cfssl was starting before nginx so nginx only ends up starting once since cfssl restarts it before it has started.